design-proposals/garbage-collect-completed-dvs.md

```diff
@@ -0,0 +1,60 @@
+# Overview
+DataVolumes are a useful construct for provisioning VM disks but once the PVC is provisioned and populated the DataVolume provides no value. This design presents an approach for garbage collecting completed DataVolumes. 
+
+## Motivation
+In the field, customers are confused by DVs after the fact. For example, when someone deletes a PVC that is owned by a DV they are confused when the PVC is re-created.
+
+## Goals
+DataVolumes will be garbage collected after their PVC is provisioned and populated.
+
+## Non Goals
+(limitations to the scope of the design)
+
+## Definition of Users
+(who is this feature set intended for)
+
+## User Stories
+* As a VM owner I use a standalone DataVolume to prepare a VM disk PVC for use by my VM. Once the DataVolume is complete, it has no use and can be confusing, so I want the system to garbage collect it. The PVC should still maintain an independent lifecycle from the VM.
+* As a VM owner I add an entry to the dataVolumeTemplates section of my VM spec to have a disk prepared for my VM. Once the DataVolume is complete, it has no use and can be confusing, so I want the system to garbage collect it. I want kubevirt to understand that it doesn't need to create a new DataVolume when the underlying PVC exists. The PVC should still share the same lifecycle as the VM.
+* As a VM owner I want to restore my backed up VM without needing to recreate the DataVolume.
```

Comment:
This is nice!

Reply:
Copied from @aglitke :)

Reply:
* As a VM owner I want to replicate my workload to another cluster without needing to mutate the PVC and DV with special annotations in order for them to behave as expected in the new cluster.

Reply:
Seems like a good point but the `special annotations` should be explicitly mentioned.